test/skate/open_route_service_api_test.exs

```diff
@@ -84,15 +84,18 @@ defmodule Skate.OpenRouteServiceAPITest do
                "steps" => [
                  %{
                    "instruction" => "not-in-results",
+                   "name" => "not-in-results",
                    # Depart
                    "type" => 11
                  },
                  %{
                    "instruction" => "1",
+                   "name" => "1",
```

Comment:
non-blocking: Could the names added here look more like street names? I was confused to see the instruction and name be the same value and had to go look up the difference between the two. Seems like `name` should be the street name?

Reply:
https://github.com/mbta/skate/compare/ae88499aacd89a6816c12859051a1afae3b29e4e..c588d63d5bd2ffa8d4ba61c0d4c68f933552519f